controller/api/public/stat_summary.go

```diff
@@ -328,18 +330,22 @@ func metricToKey(metric model.Metric, groupBy model.LabelNames) string {
 	return strings.Join(values, "/")
 }
 
-func (s *grpcServer) getMeshedPodCount(obj runtime.Object) (*meshedCount, error) {
-	pods, err := s.lister.GetPodsFor(obj)
+func (s *grpcServer) getMeshedPodCount(obj runtime.Object) (*podCount, error) {
+	pods, err := s.lister.GetPodsFor(obj, true)
 	if err != nil {
 		return nil, err
 	}
 
-	meshCount := &meshedCount{}
+	meshCount := &podCount{}
 	for _, pod := range pods {
 		meshCount.total++
 		if isInMesh(pod) {
 			meshCount.inMesh++
 		}
+
+		if pod.Status.Phase == apiv1.PodFailed {
+			meshCount.errored++
+		}
```

Comment:
i'm a little apprehensive about counting failed pods in the `total` and `inMesh` fields. specifically, when i see output like this:
```
$ bin/conduit stat -n emojivoto deploy
NAME       MESHED   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99
emoji         1/1     0.00%   0.0rps           0ms           0ms           0ms
vote-bot      1/1         -        -             -             -             -
voting        1/1     0.00%   0.0rps           0ms           0ms           0ms
web           1/1     0.00%   1.0rps           6ms          15ms          19ms
```

i expect those `MESHED` `1/1` values to indicate currently running pods (kubectl requires a `--show-all` flag to show non-running pods). if i were to see `5/6`, it's kind of ambiguous what's running/meshed/failed. i think it's fine to call out failed pods as a separate stat, but perhaps don't count them as meshed or part of total?

Reply:
yeah that makes sense. I can subtract the failed pods from the total count.